scripts/apps/authoring/views/authoring.html

```diff
@@ -10,6 +10,7 @@
                         <label translate>Modified</label>
                         <time sd-datetime data-date="item.versioncreated"></time>
                     </div>
+                    <span sd-extension-point sd-extension-point-type="authoring-sticky-detailed"></span>
```

Comment:
maybe this could be simplified `<span sd-extension-point="authoring-sticky-detailed"></span>`